### PR TITLE
test(longevity): new lwt longevity with parallel nemesis

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
Added a new longevity that uses lwt and executes parallel nemesis

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by juliayakovlev** [comment]
@bentsi  Bentsi, Roy told that this PR can be merged
@ShlomiBalalis  please, put here links to the jobs in master and 4.0. Thanks

**Comment by ShlomiBalalis** [comment]
https://jenkins.scylladb.com/view/master/job/scylla-master/job/longevity/job/longevity-lwt-24h-1Dis-2NonDis-Nemesis/
https://jenkins.scylladb.com/view/master/job/scylla-4.0/job/longevity/job/longevity-lwt-24h-1Dis-2NonDis-Nemesis/

File: test-cases/longevity/longevity-lwt-basic-24h-1dis-2nondis.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+test_duration: 1500
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml n=10000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50" ]
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=30" ]
+stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(select=1)' cl=SERIAL duration=1440m -mode native cql3 -rate threads=30" ]
+
+n_db_nodes: 4
+n_loaders: 3
+n_monitor_nodes: 1
+experimental: true
```
Comment (bentsi) [inline test-cases/longevity/longevity-lwt-basic-24h-1dis-2nondis.yaml:9]: not needed any more

Comment (ShlomiBalalis) [reply test-cases/longevity/longevity-lwt-basic-24h-1dis-2nondis.yaml]: Done

**Pull request merged**